python/cuml/test/test_incremental_pca.py

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright (c) 2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import pytest
+import cupy as cp
+
+from sklearn.decomposition import IncrementalPCA as skIPCA
+
+from cuml.datasets import make_blobs
+from cuml.decomposition import IncrementalPCA as cuIPCA
+
+from cuml.test.utils import array_equal
+
+
+@pytest.mark.parametrize('nrows', [500, 5000])
+@pytest.mark.parametrize('ncols', [15, 25])
+@pytest.mark.parametrize('n_components', [10, 12])
```

Comment:
10 and 12 components seem very similar and probably not worth testing separately. I'd probably try to get more variation, like 2 and 12 or similar